### PR TITLE
chore: fix //nolint directives

### PR DESCRIPTION
**Pull request by AlekSi**
# Pull Request

## What? (description)

chore: fix `//nolint` directives

## Why? (reasoning)

That's the recommended syntax:
https://golangci-lint.run/usage/false-positives/

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [x] you ran conformance (`make conformance`)
- [x] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

### CODE REVIEW

File: pkg/resources/config/k8s_control_plane.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ type K8sControlPlaneSchedulerSpec struct {
 
 // K8sManifestsSpec is configuration for manifests.
 //
-//nolint: malign
+//nolint:maligned
```
Comment (AlekSi) [inline pkg/resources/config/k8s_control_plane.go:78]:
```
WARN [runner/nolint] Found unknown linters in //nolint directives: malign
```

I wonder if CI can catch it

**Review by smira — APPROVED** [review]
We'll fix the CI and get this merge

**Review by Unix4ever — APPROVED** [review]

**Pull request merged**